src/sfctl/custom_service.py

```diff
@@ -212,6 +214,76 @@ def validate_activation_mode(activation_mode):
     if activation_mode not in [None, 'SharedProcess', 'ExclusiveProcess']:
         raise CLIError('Invalid activation mode specified')
 
+def parse_scaling_mechanism(scaling_mechanism):
+    """"Parse a scaling mechanism description"""
+    from azure.servicefabric.models.add_remove_incremental_named_partition_scaling_mechanism import AddRemoveIncrementalNamedPartitionScalingMechanism # pylint: disable=line-too-long
```

Comment:
You can do something like
```python
from x.y import (
    import_me
)
```
to shorten your lines. Makes it also so you don't need to disable the line too long here.

Reply:
good suggestion